houzz.com.website-services.json

```diff
@@ -0,0 +1,34 @@
+{
+	"providerId":"Houzz.com",
+	"providerName":"Houzz.com",
+	"serviceId":"website-services",
+	"serviceName":"Houzz Website Services",
+	"version": 1,
+	"logoUrl":"https://st.hzcdn.com/res/can/4231/pic/houzz_logo.svg",
+	"description":"Set up website hosting at houzz.com",
+	"syncRedirectDomain": "houzz.com",
+	"syncBlock": false,
+	"records":[
+		{
+			"groupId": "mapping",
```

Comment:
Not sure why you defined a groupId on your elements; they are all the same.  Harmless to leave in, but if this was a copy/paste oversight you could remove them.

Reply:
thx. removed `groupId`